hack/test-stirunimage.sh

```diff
@@ -136,6 +136,11 @@ docker run test >& "${WORK_DIR}/s2i-override-run.log"
 grep "Running custom run" "${WORK_DIR}/s2i-override-run.log"
 check_result $? "${WORK_DIR}/s2i-override-run.log"
 
+test_debug "s2i build with add-host option"
+s2i build https://github.com/openshift/ruby-hello-world centos/ruby-23-centos7 --add-host rubygems.org:0.0.0.0 test-ruby-app &> "${WORK_DIR}/s2i-add-host.log"
```

Comment:
since you expect this to fail you need an || true at the end, because we run our script with exit on error.

Reply:
ok, I added a `set +e` before run my test.